bin/install.sh

```
@@ -316,6 +298,17 @@ done
 
 echo "セットアップが完了しました。"
 
+# ふりがな(MeCab Ruby拡張): 同梱 vendor から構築。MeCab本体/辞書はパッケージ導入済み。
+sudo chmod 777 /usr/local/src
+cd /usr/local/src
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)